### PR TITLE
Silence healthchecks

### PR DESCRIPTION
**Pull request by binaryberry**
A few things I'd missed in PR #357, these services still show the error.

https://financialtimes.atlassian.net/jira/software/projects/NOPSCOPS/boards/952

### CODE REVIEW

File: lib/metrics/services.js [hunk 1]
```diff
@@ -178,6 +179,7 @@ module.exports = {
 	'next-eventpromo-api-datasource': /^https?:\/\/live\.ft\.com\/__service\/eventpromo\/.*/,
 	'next-magnet-api': /^https?:\/\/ft-next-magnet-api-(eu|us)\.herokuapp\.com/,
 	'next-magnet-api-email-svc': /^https?:\/\/ep\.ft\.com\/user-lists\/users\/.*\/lists/,
+	'next-magnet-api-event-promo': /^https:\/\/www.ft\.com\/eventpromo\/api.*/,
```
Comment (alexnaish) [inline lib/metrics/services.js]: Need to escape the `.` between `www` and `ft`

File: lib/metrics/services.js [hunk 2]
```diff
@@ -164,7 +165,7 @@ module.exports = {
 	'navigation-api': /next-navigation\.ft\.com/,
 	'navigation-eu-bucket': /ft-next-navigation\.s3-website-eu-west-1\.amazonaws\.com/,
 	'navigation-us-bucket': /ft-next-navigation-us\.s3-website-us-east-1\.amazonaws\.com/,
-	'navigation-origami-svc': /^https:\/\/ft\.com\/__origami\/service\/navigation\/v2\/menus/,
+	'navigation-origami-svc': /^https:\/\/www.ft\.com\/__origami\/service\/navigation\/v2\/menus/,
```
Comment (alexnaish) [inline lib/metrics/services.js]: Need to escape the dot here + is the www actually required? Could make it optional?

File: lib/metrics/services.js [hunk 3]
```diff
@@ -94,9 +94,10 @@ module.exports = {
 	'ft-next-comments-api': /^https?:\/\/comments-api\.ft\.com/,
 	'ft-next-api-user-prefs-v002': /^https?:\/\/ft-next-api-user-prefs-v002\.herokuapp\.com/,
 	'ft-next-article-preview': /^https?:\/\/ft-next-article-(eu|us)\.herokuapp\.com\/preview/,
-	'ft-next-enterprise-tools': /^https?:\/\/enterprise-tools-us\.herokuapp\.com\/__data\/.*/,
-	'ft-next-enterprise-tools-membership-auth-svc': /^https?:\/\/api-authz-svc\.memb\.ft\.com\api\/v2\/\authorize/,
-	'ft-next-enterprise-tools-b2b-fulfil-svc': /^https?:\/\/b2b-fulfil-svc-eu-prod\.memb\.ft\.com\/personalisedInvitations/,
+	'ft-next-enterprise-tools': /^https?:\/\/enterprise-tools-(eu|us)\.herokuapp\.com\/__data\/.*/,
+	'ft-next-enterprise-tools-membership-auth-svc': /^https?:\/\/api-authz-svc\.memb\.ft\.com\/api\/v2\/authorize/,
+	'ft-next-enterprise-tools-b2b-fulfil-svc': /^https?:\/\/b2b-fulfil-svc-eu-prod\.memb\.ft\.com\/personalisedInvitations\/.*/,
+	'ft-next-enterprise-tools-api-ft-com': /^https?:\/\/api\.ft\.com\/idm\/v1\/password-token/,
```
Comment (alexnaish) [inline lib/metrics/services.js]: Since this API endpoint isn't actually specific to ET, could we call it something more generic?

Comment (binaryberry) [reply lib/metrics/services.js]: Thanks! Amended in `e7e2510`, as well as a few other ones. In my previous PR I didn't realise that these checks apply across all apps and should therefore only be named after the endpoint they call to, not from.

File: lib/metrics/services.js (hunk 2 above)
Comment (binaryberry) [reply lib/metrics/services.js]: Thanks, made www optional in 337fbf4. While escaping the dot I realised a lot of other dots needed escaping so I also rectified them in adc2d144 .

File: lib/metrics/services.js (hunk 1 above)
Comment (binaryberry) [reply lib/metrics/services.js]: Thanks, rectified in adc2d14.

**Pull request merged**